sparklightautoml/dataset/persistence.py

```diff
@@ -207,7 +223,7 @@ def __init__(self,
         self._no_unpersisting = no_unpersisting
 
     def _persist(self, pdf: PersistableDataFrame, level: PersistenceLevel) -> PersistableDataFrame:
-        spark = SparkSession.getActiveSession()
+        spark = pdf.sdf.sql_ctx.sparkSession
```

Comment:
accessing through internal structures look hackish, function for accessing spark session could be better